test/helpers/envtest.go

```diff
@@ -278,3 +280,32 @@ func (t *TestEnvironment) Stop() error {
 	t.cancel()
 	return t.env.Stop()
 }
+
+func getFilePathToCAPICRDs(root string) string {
+	modBits, err := os.ReadFile(filepath.Join(root, "go.mod")) //nolint:gosec
+	if err != nil {
+		return ""
+	}
+
+	var clusterAPIVersion string
+	for _, line := range strings.Split(string(modBits), "\n") {
+		matches := clusterAPIVersionRegex.FindStringSubmatch(line)
+		if len(matches) == 3 {
+			clusterAPIVersion = matches[2]
+		}
+	}
+
+	if clusterAPIVersion == "" {
+		return ""
+	}
+
+	gopath := envOr("GOPATH", build.Default.GOPATH)
+	return filepath.Join(gopath, "pkg", "mod", "sigs.k8s.io", fmt.Sprintf("cluster-api@v%s", clusterAPIVersion), "config", "crd", "bases")
```

Comment:
I suggest taking a look at [runtime.Caller](https://pkg.go.dev/runtime#Caller). I think it returns just what you need here.

Reply:
Although that might not help since you need the stuff from config crd bases. And you would have to manipulate the path nevertheless. But maybe it helps you eliminate the need for GOPATH check. 🤷  I leave it up to you. :D

Reply:
This copied from CAPZ...i didn't change it. But we do already use `runtime.Caller` to get the root: https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/e0fc4de3bc5f4008b214a8cab6c72a5fe51c6960/test/helpers/envtest.go#L85

Reply:
Oh nice. :D ok. :)